src/main/java/io/confluent/connect/jdbc/dialect/DatabaseDialect.java

```diff
@@ -425,6 +426,26 @@ void bindField(
       Object value
   ) throws SQLException;
 
+  /**
+   * Method that binds a value with the given schema at the specified variable within a prepared
+   * statement.
+   *
+   * @param statement the prepared statement; may not be null
+   * @param index     the 1-based index of the variable within the prepared statement
+   * @param schema    the schema for the value; may be null only if the value is null
+   * @param value     the value to be bound to the variable; may be null
+   * @param colDef    the Definition of the column to be bound; may be null
+   * @throws SQLException if there is a problem binding the value into the statement
+   * @see #statementBinder
+   */
+  void bindField(
```

Comment:
This is a backwards incompatible change, and anyone not inheriting from GenericDatabaseDialect won't have an implementation of this method.